mono/mini/mini-wasm-debugger.c

```diff
@@ -588,6 +597,16 @@ typedef struct {
 static gboolean describe_value(MonoType * type, gpointer addr)
 {
 	ERROR_DECL (error);
+
+	if (!datetime_class) {
+		datetime_class = mono_class_from_name_checked (mono_get_corlib(), "System", "DateTime", error);
+	}
```

Comment:
These should use the GENERATE_TRY_GET_CLASS_WITH_CACHE macro:

static GENERATE_TRY_GET_CLASS_WITH_CACHE (fixed_buffer, "System.Runtime.CompilerServices", "FixedBufferAttribute")

Reply:
update in recent commits